components/x-gift-article/src/lib/highlightsApi.js

```diff
@@ -54,4 +56,19 @@ export default class HighlightsApiClient {
 			return { annotationsCopyResult: undefined }
 		}
 	}
+
+	async shareHighlights(articleId,includeHighlights = false) {
+		try {
+			if(!includeHighlights){
+				return{};
+			}
+			return await this.fetchJson('/create-token', {
+				method: 'POST',
+				body: JSON.stringify({ articleId })
+			})
+
+		} catch (error) {
+			return{};
+		}
+	}
```

Comment:
**Question / change request** Has linting correctly been run on these changes? I would have expected some linting errors here based on this projects eslint config.

I would have expected the `keyword-spacing` rules to fail here on stuff like `if(` and `return{}`

Reply:
If you run linting on this and it passes then fine to leave it but if it fails then please update it

Reply:
hmmm , yeah you are right , for some reason the pre-commit script didn't run , take a look at this